blocks/library/image/style.scss

```diff
@@ -15,6 +15,11 @@
 		float: right;
 		margin-left: 15px;
 	}
+
+	&[data-align="full"] {
+		width: 90%;
```

Comment:
It was a number pulled out of the air 😄  But yes, specifically it's a value close to 100% while still allowing space for the block mover.